doc/sphinx/source/tutorial.rst

```diff
@@ -195,22 +163,22 @@ again. Let's dive into an example right away:
 
 .. code-block:: lua
 
-   use("aux", "coroutine")
+   include "scripting/coroutine.lua"
 
-   function print_a_word(word)
-      while 1 do
+   function print_a_word(word)         -- this function is a coroutine
```

Comment:
Strictly this is not correct (and the paragraph above is also overly simplistic). `print_a_word` is a *function*. The call to `run(f)` then creates a new coroutine object from the function `f`. A coroutine, in Widelands's Lua implementation, is a wrapper *object* around a function in a *Lua State*, with its own *Lua stack* and set of local variables. Most of the time this is a fine distinction, but when you want more control over your script's workflow it becomes a very important one.
We don't need to explain it in such detail in the beginner's tutorial but we shouldn't give incorrect information either.

Reply:
" -- this function will get a coroutine" ?

Reply:
"get" might be interpreted as "receive" in this context. It's a word with many different meanings which we're overfond of using when translating German into English ;D

"Function from which a coroutine will later be constructed" or "A function we'll use to create a coroutine" would be clearer imho. 
This could be clarified further by changing the comment on the `run(...)` line to "Constructs a new coroutine from the function `print_a_word` and an argument, and immediately launches this coroutine."

Reply:
Good explanation :+1: 

Done